### ISSUE TITLE
Fix HTML refresh generated by the captive portal daemon.

### ISSUE THREAD

**Issue by virtadpt** [Byzantium/Byzantium#222]
One of the problems with web browsers is that they cache content, which also means that redirects don't always work because they want to hang onto the original page.

**Comment by virtadpt** [comment]
stackoverflow.com/questions/1341089/using-meta-tags-to-turn-off-caching-in-all-browsers

**Comment by virtadpt** [comment]
Here is the tcpdump evocation I used during testing:

```
tcpdump -i wlan0 -s 0 -v -X -n port 53 or port 80 or port 443
```

**Comment by virtadpt** [comment]
Fixed in commit 150a8d1b31aef6fadda5353c665d46a0b9966a78.
